Update diagram generation docs (#39)
anz-bank/sysl-website: docs/cmd-integrations.md

sidebar_label: Integration Diagram
---

:::info

> We are currently in the process of migrating from PlantUML to Mermaid for our diagram generation. This will remove the external dependency on PlantUML and offer a better user experience. Diagram generation with mermaid is currently supported for integration diagrams and sequence diagrams only. For more details, check out [sysl diagram](cmd-diagram)

Reviewer: ```suggestion
We are currently in the process of migrating from PlantUML to Mermaid for our diagram generation. This will remove the external dependency on PlantUML and offer a better user experience. Diagram generation with Mermaid is currently supported for integration diagrams and sequence diagrams only. For more details, check out [sysl diagram](cmd-diagram).
```

:::

:::info

> This command requires the SYSL_PLANTUML environment variable to be set or passed in as a flag. Follow the instructions [here](plantuml.md) for more details

Reviewer: ```suggestion
This command requires the SYSL_PLANTUML environment variable to be set or passed in as a flag. Follow the instructions [here](plantuml.md) for more details.
```

:::
---

> `sysl integrations` lets you generate integration diagrams. The command requires a project to be specified to produce an integration diagram. Refer to the examples for more details.

Reviewer: ```suggestion
`sysl integrations` generates integration diagrams. The command requires a project to be specified to produce an integration diagram. Refer to the examples for more details.
```

> - ints `usage: sysl ints [<flags>] <MODULE>`

Reviewer 2: ```suggestion
```bash
sysl ints [<flags>] <MODULE>
```
```

## Output Formats

> The output file format can be specified via the extension passed into the -o flag.
> Valid formats include .svg, .png, .uml, .puml, .plantuml, .html or .link

Reviewer: ```suggestion
The output file format can be specified via the extension of the filename passed into the `-o` flag.

Valid extensions include `.svg`, `.png`, `.uml`, `.puml`, `.plantuml`, `.html` or `.link`.
```
